Author: Andrea Veri <and@debian.org>
Description: Use AM_MAINTAINER_MODE, to avoid re-running autotools 
 except when explicitely called in debian/rules.

diff -urNad agg-2.5+dfsg~/configure.ac agg-2.5+dfsg/configure.ac
--- agg-2.5+dfsg~/configure.ac 2007-07-21 19:58:06.966030999 -0500
+++ agg-2.5+dfsg/configure.ac  2007-07-21 19:58:33.595695250 -0500
@@ -2,7 +2,7 @@
 AC_CANONICAL_TARGET
 AC_CONFIG_HEADERS(include/config.h)
 AM_INIT_AUTOMAKE(agg, 2.4.0)
-
+AM_MAINTAINER_MODE
 
 dnl Checks for programs.
 AC_PROG_CC
